rest/taskana-rest-spring/src/test/java/pro/taskana/rest/TaskControllerIntTest.java

```diff
@@ -421,20 +427,20 @@ void testThrowsExceptionIfInvalidFilterIsUsed() {
   void testGetLastPageSortedByPorValue() {
 
     HttpEntity<String> request = new HttpEntity<>(restHelper.getHeadersAdmin());
-    ResponseEntity<TaskSummaryListResource> response =
+    ResponseEntity<TaskanaPagedModel<TaskSummaryRepresentationModel>> response =
         template.exchange(
             restHelper.toUrl(Mapping.URL_TASKS)
                 + "?state=READY,CLAIMED&sort-by=por.value&order=desc&page=15&page-size=5",
             HttpMethod.GET,
             request,
-            ParameterizedTypeReference.forType(TaskSummaryListResource.class));
-    assertThat(response.getBody().getContent()).hasSize(1);
+            TASK_SUMMARY_PAGE_MODEL_TYPE);
+    assertThat(Objects.requireNonNull(response.getBody()).getContent()).hasSize(1);
```

Comment:
Please don't do this. 
Add `assertThat(response.getBody()).isNotNull()`
You have done this 16 times. Please replace this everywhere.